extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -283,12 +300,14 @@ export const BlockaidAssetWarning = ({
       <div>
         <div className="ScamAssetWarning__description">
           {t(
-            "This token was flagged as malicious by Blockaid. Interacting with this token may result in loss of funds and is not recommended for the following reasons",
+            `This token was flagged as ${
```

Comment:
Does this text show up for Benign scans as well?
Also I don't think `result_type` is optional so not sure if we need the fallback case to `malicious` here.

Reply:
This does not show up for Benign because we check for non-benign before we instantiate this component

Reply:
Yeah, good point about `result_type` not being optional. I'm probably being overly cautious here